WooCommerce/Classes/ViewRelated/Orders/OrdersRootViewController.swift

```diff
@@ -203,18 +203,25 @@ final class OrdersRootViewController: UIViewController {
     /// Presents the Order Creation flow when a product is scanned
     ///
     @objc func presentOrderCreationFlowByProductScanning() {
+        ServiceLocator.analytics.track(event: WooAnalyticsEvent.Orders.orderAddNewFromBarcodeScanningTapped())
+
         guard let navigationController = navigationController else {
             return
         }
 
         let productSKUBarcodeScannerCoordinator = ProductSKUBarcodeScannerCoordinator(sourceNavigationController: navigationController,
                                                                                       onSKUBarcodeScanned: { [weak self] scannedBarcode in
+            ServiceLocator.analytics.track(event: WooAnalyticsEvent.Orders.barcodeScanningSuccess(from: .orderList))
```

Comment:
```suggestion
            analytics.track(event: WooAnalyticsEvent.Orders.barcodeScanningSuccess(from: .orderList))
```

We have a reference to `ServiceLocator.analytics` in the class declaration, we can remove this one and reference the existing one. Added to this, perhaps we should inject it as well in the init?

Reply:
Thanks for the comment! Indeed, it makes sense to change it, I did it in 13fa2b7ddc685c707e8979a28c1b73787d0dc996 

> Added to this, perhaps we should inject it as well in the init?

I think for now as we don't have tests for this view controller we can keep it without being injected, but if the requirements change we can do it 👍